ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.view.js

```diff
@@ -216,13 +236,15 @@ const ResultItemView = Marionette.LayoutView.extend({
   },
   tagName: CustomElements.register('result-item'),
   modelEvents: {},
+  itemSelected: false,
```

Comment:
❓ What is `itemSelected` for?

Reply:
Flag set when users select/click the item in the list. Render method uses this to determine whether the thumbnail should be loaded for this item.

Reply:
❗️ Gotcha, if that's the case we should be leveraging the selectionInterface for a few reasons.

One is that the results can be selected from multiple areas, including the map or the histogram, which this would miss.  

Another is that results can be selected in groups in this list by using shift click, which this would also miss.

Reply:
I removed list view thumbnail rendering completely. Also refactored the result indicator region out.